tests/lac/diagonal_matrix_02.with_trilinos=true.with_mpi=true.with_p4est=true.mpirun=7.output

```diff
@@ -0,0 +1,13 @@
+
+DEAL:2d::Testing FE_Q<2>(1)
+DEAL:2d::Ok.
+DEAL:2d::
+DEAL:2d::Testing FE_Q<2>(2)
+DEAL:2d::Ok.
+DEAL:2d::
+DEAL:3d::Testing FE_Q<3>(1)
+DEAL:3d::Ok.
+DEAL:3d::
+DEAL:3d::Testing FE_Q<3>(2)
+DEAL:3d::Ok.
+DEAL:3d::
```

Comment:
i will switch the test to 1 global + 1 adaptive refinement case in 2D so that this unit test makes sense.